Makefile

```diff
@@ -11,7 +11,7 @@ test:
 	dune runtest
 
 test-all:
-	dune build @runtest-all
+	dune runtest --non-deterministic
```

Comment:
Shouldn't this be `--profile=non-deterministic`?

Reply:
It should, let me fix that!